src/test/java/org/broadinstitute/hellbender/utils/samples/SampleDBUnitTest.java

```diff
@@ -111,6 +111,15 @@ public void loadPEDString() {
         Assert.assertEquals(testPEDSamplesAsSet, db.getSamples());
     }
 
+    @Test()
+    public void loadPEDFromNames() {
```

Comment:
Test method should be named after the method it's testing -- eg., `testAddSamplesFromSampleNames()`

Reply:
changed; however, that same comment could apply to virtually every other test in SampleDBUnitTest (all preexisting)